src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -157,15 +157,14 @@ def register_common_storage_account_options(self):
                           validator=validate_encryption_services, help='Specifies which service(s) to encrypt.')
 
     def register_precondition_options(self):
-        from ._validators import validate_match_condition
         self.extra('if_modified_since')
         self.extra('if_unmodified_since')
         self.extra('if_match', help="An ETag value, or the wildcard character (*). Specify this header to perform the "
                    "operation only if the resource's ETag matches the value specified.")
         self.extra('if_none_match', help="An ETag value, or the wildcard character (*). Specify this header to perform "
                    "the operation only if the resource's ETag does not match the value specified. Specify the wildcard "
                    "character (*) to perform the operation only if the resource does not exist, and fail the operation "
-                   "if it does exist.", validator=validate_match_condition)
```

Comment:
We don't need this validator now?

Reply:
it is used to transform Etag to match condition. From investigation, track2 can accept them as keyword.
For test, you can see 
https://github.com/Azure/azure-cli/blob/5fce7f550cf7e201c4c943608c4a0ecd0cb6d62b/src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_oauth_track2.py#L166-L186